packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,281 @@
+# Nimble Select (IxD)
+
+## Overview
+
+The select component is used to present a pre-defined list of options that end-users can choose from. Selects simplify choice-making, reduce errors, and conserve screen space for more efficient and user-friendly interactions.
+
+> NOTE: THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+
+-   [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+-   [ViD Select Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=2597-83231&mode=design)
+
+## Usage
+
+When to use:
+
+-   To present a pre-defined list of options
+-   For filtering or sorting data in tables or lists
+-   For selecting settings or preferences
+-   For single selection
+
+When not to use:
+
+-   To allow end-users to add custom options
+-   When there are fewer than three options
+-   For complex and/or hierarchical data
+-   For multi selection
+
+### Anatomy
+
+![ ](spec-images/select-1.png)
+
+| Element      | Description                                                             |
+| ------------ | ----------------------------------------------------------------------- |
+| Label        | Text that tells the end-user what to expect in the list of options      |
+| Field button | Click area that opens the menu and displays the current selected option |
+| Menu         | A list of options to choose from displayed as an _open_ state           |
+| Menu item    | A selection the end-user can make, shown with other options in the menu |
+
+#### Label
+
+Selects should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases).
+
+Keep the label text as concise as possible, limited to a single line of text. Use sentence casing.
+
+In a form, use the label to indicate whether the select is optional or required for the end-user to interact with. If most of the components in the form are required, indicate which components are optional. If most of the components in the form are optional, indicate which components are required. Required or optional indication in the label text should always follow the pattern "[Label] (optional)" or "[Label] (required)".
+
+![ ](spec-images/select-2.png)
+
+#### Field Button
+
+Selects can be empty by default or have a default selection from the menu. Empty selects should include placeholder text that propts the end-user to select an option from the menu. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Ellipses are not needed. Use sentence casing.
```

Comment:
When a select is empty by default, do we need a way for the user to be able to re-enter that default state after they've made a selection? Does that answer change based on whether filling in a value is optional or required?

Reply:
Except I'm not sure how you have an "optional" select when it requires a default value?

Reply:
I suspect we will get requests for a special unset value. A classic use case is picking a user (e.g. assigning a work item owner in azdo) or a state/province/country where it's hard to pick a good default and you want to force the user to select something rather than potentially skip over the field and leave it at a bad default value.

If we get rid of the empty/placeholder state then I'm not sure what we'd tell those clients. Use a combobox? But we don't want to let them invent a state/country by typing in an arbitrary value.

In my experience with selects which allow an unset value, some of them never let you unset it after having set it and some of them do let you unset it, eg. by deleting all the text in the azdo assigned to field, (which is more similar to a combobox).

It's a sneaky problem. My vote would be either to design an answer for unset values in the select now or move it to Future Considerations, but not to forever disallow it.

Reply:
I've simply added the guidance that the empty state should only be used when there are no obvious default list options.